examples/simulate_in_parallel.py

```diff
@@ -1,21 +1,20 @@
 from pathlib import Path
-from quantinuum_schemas.models.backend_config import AerConfig
+from quantinuum_schemas.models.backend_config import AerConfig, QulacsConfig
 from pytket.qasm.qasm import circuit_from_qasm
-from tierkreis.graphs.aer.compile_run import aer_compile_run
+from tierkreis.graphs.simulate.compile_run import compile_run
 from uuid import UUID
 
 from tierkreis.consts import PACKAGE_PATH
 from tierkreis.controller import run_graph
 from tierkreis.storage import FileStorage, read_outputs
 from tierkreis.executor import UvExecutor
 
-config = AerConfig()
 circuit = circuit_from_qasm(Path(__file__).parent / "data" / "ghz_state_n23.qasm")
 circuits = [circuit] * 10
 
 
-def main():
-    g = aer_compile_run()
+def main(config: AerConfig | QulacsConfig):
```

Comment:
I think calling this "main" is perhaps a bit unclear, I would not expect "main" to have any arguments, perhaps it is worth re-naming this function to "simulate_example" and have different configs passed in a different "main" function?